docs/content/latest/explore/ysql-language-features/queries.md

```diff
@@ -510,10 +510,10 @@ SELECT employee_no FROM employees GROUP BY employee_no;
 
 If there were duplicate rows in the result set from the preceding example, these rows would have been removed because the `GROUP BY` clause functions like the `DISTINCT` clause in this case.
 
-The following example shows how to select the total amount that each employee has been paid. The `GROUP BY` clause divides the rows in the `employees` table into groups by employee number. For each group, the total amounts are calculated using the `SUM()` function:
+If the `employees` table had a column called `amount` that contained the employee pay data, the following example would have shown how to select the total amount that each employee has been paid. The `GROUP BY` clause would have divided the rows in the `employees` table into groups by employee number. For each group, the total amounts would have been calculated using the `SUM()` function:
```

Comment:
Too much conditional in here. Can you rework it so it actually _does_ show the things you say it _would have_ shown? And show sample output? (At which point, the previous wording is probably good...)
```suggestion
If the `employees` table had an `amount` column that contained the employee pay data, the following example would have shown how to select the total amount that each employee has been paid. The `GROUP BY` clause would have divided the rows in the `employees` table into groups by employee number. For each group, the total amounts would have been calculated using the `SUM()` function:
```

Reply:
Reworded slightly, but don't have time for a new example setup right now. Will do this some other time (made a note for myself)